docs/stripe_elements_js.rst

```diff
@@ -0,0 +1,48 @@
+A note on Stripe Elements JS methods
+====================================
+
+.. note::
+  TLDR: If you haven't yet migrated to PaymentIntents,
+  prefer ``stripe.createSource()`` over ``stripe.createToken()`` for better compatibility with PaymentMethods.
+
+
+A point that can cause confusion when integrating Stripe on the web is that there
+are multiple generations of frontend JS APIs that use Stripe Elements with stripe js v3.
+
+In descending order of preference these are:
+
+Payment Intents (SCA compliant)
+-------------------------------
+
+The newest and preferred way of handling payments, which supports SCA compliance (3D secure etc).
+
+See https://stripe.com/docs/payments/payment-intents/web
+
+
+Charges using stripe.createSource()
+-----------------------------------
+
+This creates Source objects within Stripe, and can be used for various different
+methods of payment (including, but not limited to cards), but isn't SCA compliant.
```

Comment:
> isn't SCA compliant
a bit more elaboration here would help. this seems like using createSource is discouraged